LukeMason33/Rotten-Tomatillos
# Feature header components

### Is this a fix or a feature?

Feature.

### What is the change/fix?

Renames some `/header-component` files, add `/filter-menu-component` and `search-bar-component` folders with related js and scss files.

### Where should the reviewer start?

Familiarize yourself with the component functions added to the `/header-component` folder. Review `changeHandler` function on `App.js`, and how it is being used to set the state of filteredMovies. Review the conditional rendering in `App.js` rendering.


### How should this be tested?

In browser, search for an existing movie in the search bar and ensure that it is filtered/displayed appropriately. Also check that no movies are displayed if search requirements aren't met.
